packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -117,11 +115,10 @@ class SimpleFluidError implements IFluidErrorBase {
 /** For backwards compatibility with pre-fluidErrorCode valid errors */
 function patchWithErrorCode(
     legacyError: Omit<IFluidErrorBase, "fluidErrorCode">,
-    errorCode: string = "<error predates fluidErrorCode>",
 ): asserts legacyError is IFluidErrorBase {
-    const patchMe: { fluidErrorCode?: string } = legacyError as any;
+    const patchMe: { -readonly [P in "fluidErrorCode"]?: IFluidErrorBase[P] } = legacyError as any;
     if (patchMe.fluidErrorCode === undefined) {
-        patchMe.fluidErrorCode = errorCode;
+        patchMe.fluidErrorCode = "<error predates fluidErrorCode>";
```

Comment:
this the wrong format for fluid error code. shouldn't we just not add the property

Reply:
in my mind the existence of the property or not may be important. so we should never add it beyond initial creation

Reply:
WF -- This is a temporary measure to backfill the presence of the property in cross-version scenarios.  I'm not going to change it now, it will be phased out in a few more months anyway and is useful to be able to search for this string to watch for when we can do so.